cl/users/email_handlers.py

```diff
@@ -676,3 +677,83 @@ def enqueue_email(recipients: list[str], message_id: str) -> None:
             f"The message: {message_id} can't be enqueued because it "
             "doesn't exist anymore."
         )
+
+
+PERMANENT = {
+    EMAIL_NOTIFICATIONS.GENERAL: "Your email provider sent a hard "
+    "bounce message indicating we can't continue sending messages to your "
+    "current email address. "
+    "Please consider changing your email address.",
+    EMAIL_NOTIFICATIONS.NO_EMAIL: "Your email provider sent a bounce "
+    "message indicating that your email address doesn't exist. Please "
+    "consider changing your email address.",
+    EMAIL_NOTIFICATIONS.COMPLAINT: "We've received a complaint from "
+    "your email provider indicating that you don't want to receive emails "
+    "from us. Please consider changing your email address.",
+}
+
+TRANSIENT = {
+    EMAIL_NOTIFICATIONS.GENERAL: "Your email provider sent a bounce message. "
+    "We are still attempting to fix this problem by resending "
+    "messages.",
+    EMAIL_NOTIFICATIONS.MAILBOX_FULL: "Your email provider sent a "
+    "bounce message because your inbox was full. Please consider clearing out "
+    "your email inbox.",
+}
+
```

Comment:
I think these are much nicer messages for our end user.  Good job.

I think we could still workshop the transient.general message to be more colloquial.

Reply:
Sure, I changed that message to: `The last messages we sent to your email address have bounced back. We are still attempting to fix this problem by resending messages.` 
let me know what you think.